### ISSUE TITLE
frequency sampling

### ISSUE THREAD

**Issue by GvdDool** [Ouranosinc/xclim#1310]
### Setup Information

Xclim version: 0.40.0
Python version: 3.9.13
Operating System: Windows - Jupyter notebook

### Context

Hi Team, I am still fairly new to this code repository and working with GitHub in a larger team, but when looking at the Consecutive Dry Days (which is a family member in the branch _threshold), I noticed that it is possible to define the frequency much more specific than the documentation shows. For example: Q(S)-MAR: Quarter frequency, anchored at the end (or beginning) of March, works, while the xclim documentation only suggests "YS" (as the start of the year)

It seems that the function is a wrapper around the TimeGrouper function, below code is generated when an unsupported frequency is passed in the CDD function:
```
File ~\anaconda3\envs\geo_py\lib\site-packages\pandas\core\resample.py:1587, in TimeGrouper.__init__(self, freq, closed, label, how, axis, fill_method, limit, loffset, kind, convention, base, origin, offset, group_keys, **kwargs)
   1584 if convention not in {None, "start", "end", "e", "s"}:
   1585     raise ValueError(f"Unsupported value {convention} for `convention`")
-> 1587 freq = to_offset(freq)
   1589 end_types = {"M", "A", "Q", "BM", "BA", "BQ", "W"}
   1590 rule = freq.rule_code
```

There are many different frequency groupings possible, see:
https://docs.xarray.dev/en/stable/generated/xarray.cftime_range.html

Would it be a good idea to link to these options in the documentation, or is "YS" selected for a specific reason?

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by aulemahal** [comment]
Indeed, the resampling frequencies are short codes that might be unfamiliar for many users. Linking from the doc to some page explaining all this jargon is a good idea. The one you linked seems a very good candidate as it also explains the differences of what `xarray/cftime` supports.

The "YS" choice was simply that it seemed the most common and probable choice for most cases. 

To avoid cluttering the documentation in the python file and the function themselves, a compromise solution would be to add a line with a link in the automatically-generated indicator documentation. Something to be done in parallel to #1305 (see second comment).

**Comment by GvdDool** [comment]
Hi @aulemahal (Pascal), 
thanks for picking this up, I think that the error code is fine (as this is generated by the underlying (original) code, but it could be an idea to add the link, with a short description, to the documentation, so that it is clear that other frequencies are possible. I read the description and was not sure if "YS" was the only supported frequency grouper, so I did some experimenting with the function, as my target area has two seasons: JJAS and DJFM and came up with two maps, one monthly (MS) with 5mm and one yearly (YS) with 10mm to illustrate the different rainfall patterns; QS-JUN/QS-JUL would have been nice, but then I am missing one month, but others might have "true seasonality, and then it could be interesting to let the frequency start at the beginning of a season. 

I would, if possible, Highlight QS-JUN/QS-JUL, as an example, which could be perhaps a nice addition to the documentation.

**Comment by GvdDool** [comment]
> in parallel to #1305 [https://github.com/Ouranosinc/xclim/issues/1305](url) (see second comment).
In defence of the developer, the wildfire index is a complex index with multiple input, so I am not surprised that the documentation is a bit unclear - I used the code to compute kbdi earlier and had a look at the documentation for the cffwis_indices. Especially for this index, there seem to be multiple nodes, and the code resides at two locations (if I am not mistaken), once under atmos and a second part under Fire indices submodule.  

Happy to review the documentation if this is being reworked (unfortunately, I can't take the lead due to other commitments)

**Comment by Zeitsperre** [comment]
@aulemahal Has this been addressed?

**Comment by aulemahal** [comment]
I believe not.
